source/Preferences.cpp

```diff
@@ -37,6 +37,9 @@ namespace {
 	const string EXPEND_AMMO = "Escorts expend ammo";
 	const string FRUGAL_ESCORTS = "Escorts use ammo frugally";
 
+	const vector<string> DATEFMT_OPTIONS = {"DMY", "MDY", "YMD"};
```

Comment:
These are the strings that actually appear in the preferences, hence why one of my suggestions earlier changed them to display like this.
```suggestion
	const vector<string> DATEFMT_OPTIONS = {"dd/mm/yyyy", "mm/dd/yyyy", "yyyy-mm-dd"};
```
![image](https://github.com/endless-sky/endless-sky/assets/17688683/0fbb107e-6781-41c7-b5d9-2381a821174c)